src/onnx/onnx.cpp

```diff
@@ -490,6 +490,16 @@ struct onnx_parser
         {
             axis = parse_value(attributes.at("axis")).at<int>();
         }
+
```

Comment:
Can we make a function that adds a contiguous to the program if the arg isn't in standard shape? This can be done for parse_reshape() as well.

Reply:
fixed